pkgs/applications/video/ani-cli/default.nix

```diff
@@ -4,12 +4,24 @@
 , lib
 , gnugrep
 , gnused
-, wget
+, curl
 , fzf
 , mpv
+, vlc
+, iina
+, catt
+, syncplay
+, ffmpeg
 , aria2
+, player ? "mpv"
```

Comment:
Please follow the `withMpv` schemes in https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/audio/soundkonverter/default.nix.

Reply:
I looked it up and saw that they also but the runtime deps `buildInputs` I thought you are not supposed to do that?

Reply:
No. I guess it's up to them to clean up the package.